src/cc/tests/graph_test.cc

```diff
@@ -358,14 +358,18 @@ TEST_P(StorageTypeGraphTest, NodeFeaturesMultipleNodesSingleFeature)
     m.m_nodes.push_back(TestGraph::Node{.m_id = 1, .m_type = 1, .m_weight = 1.0f, .m_float_features = f2});
     auto path = std::filesystem::temp_directory_path();
     TestGraph::convert(path, "0_0", std::move(m), 2);
-    snark::Graph g(path.string(), std::vector<uint32_t>{0}, GetParam(), "");
-    std::vector<snark::NodeId> nodes = {0, 1};
-    std::vector<uint8_t> output(4 * 3 * 2);
-    std::vector<snark::FeatureMeta> features = {{0, 12}};
 
-    g.GetNodeFeature(std::span(nodes), std::span(features), std::span(output));
-    std::span res(reinterpret_cast<float *>(output.data()), output.size() / sizeof(float));
-    EXPECT_EQ(std::vector<float>(std::begin(res), std::end(res)), std::vector<float>({1, 2, 3, 5, 6, 7}));
+    for (size_t i = 0; i < 2; i++)
+    {
+        snark::Graph g(path.string(), std::vector<uint32_t>{0}, GetParam(), "", i % 2 == 0);
```

Comment:
Please, expand the existing parameterized tests: https://google.github.io/googletest/advanced.html#value-parameterized-tests

Reply:
Thanks a lot, I followed the guide and updated the tests.